source/brailleDisplayDrivers/albatross/driver.py

```diff
@@ -251,16 +263,8 @@ def _initPort(self, i: int = MAX_INIT_RETRIES - 1) -> bool:
 				return False
 			return True
 		except IOError:
-			if i == MAX_INIT_RETRIES - 1:
-				log.debug(f"Port {self._currentPort} not initialized", exc_info=True)
-				return False
-			log.debug(
-				f"Port {self._currentPort} not initialized, sleeping {SLEEP_TIMEOUT} seconds "
-				f"before try {i + 2} / {MAX_INIT_RETRIES}",
-				exc_info=True
-			)
-			time.sleep(SLEEP_TIMEOUT)
-			return False
+			log.debug(f"Port {self._currentPort} not initialized", exc_info=True)
+			return None

```

Comment:
instead of returning None why not re-raise the error?

I think  `_initPort` and it's caller could be simplified by:
- remove the try-except for `_initPort`. Let the IOError be raised. Just keep the stuff that was in the try block
- Note in the docstring that `IOError` can be raised
- Change the return type to `bool` instead of `Optional[bool]`